src/js/tooltip.js

```diff
@@ -28,7 +28,7 @@ class Tooltip {
 		this.visible = false;
 
 		this.delegates = {
-			target: new Delegate(this.target.targetEl),
+			target: new Delegate(document.getElementById(this.opts.target)),
```

Comment:
@JakeChampion I didn't realise `Target` was being stubbed and so hard to restore it; tests were passing when I did a `document.getElementById` but not when I lacked it. Now realising that `Target` is stubbed, I restored it and have reverted to the previous code; see updated commit.